src/main/java/seedu/contax/logic/commands/FindByTagCommand.java

```diff
@@ -29,6 +29,13 @@ public FindByTagCommand(TagNameContainsKeywordsPredicate predicate) {
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.updateFilteredPersonList(predicate);
+
+        // If tag does not exist in the list, just show that there are no persons
+        if (!predicate.existsInTagList(model.getFilteredTagList())) {
+            return new CommandResult(String.format(Messages.MESSAGE_PERSONS_LISTED_OVERVIEW, 0),
+                    GuiListContentType.PERSON);
+        }
```

Comment:
This is redundant, if you really want to do a defensive mechanism, add a always false predicate in here.

Reply:
Fixed in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/324/commits/b6d8de73f1cb326c8b411ffc056e66df4c0d51cf